.github/workflows/mirror-sphinx-shared-resources.yml

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+---
+name: Mirror sphinx-shared-resources
+
+# This workflow runs when a PR gets merged into the main branch and includes
+# changes to any files in ferrocene/doc/sphinx-shared-resources/.
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "ferrocene/doc/sphinx-shared-resources/**"
+  pull_request:
```

Comment:
I know this is used for testing, leaving a comment as a reminder to remove this line before we merge.